.github/workflows/build_all.yml

```diff
@@ -41,9 +44,9 @@ jobs:
       # Or is branch name when on release branch
       - name: Set Helm chart version
         run: |
-          if [ "${GITHUB_REF##*/}" = "master" ] ;then
+          if [ "${GITHUB_REF}" = "refs/heads/main" ] ;then
```

Comment:
I don't think magma has a "main" branch?

Reply:
you're right